desktop/src/views/Providers/ProviderCard.tsx

```diff
@@ -219,6 +219,7 @@ export function ProviderCard({ id, provider, remove }: TProviderCardProps) {
                     })
                   }
                   icon={<Icon as={HiDuplicate} boxSize="4" />}
+                  isDisabled={provider.isProxyProvider}
```

Comment:
just a question - so Proxy Provider == Pro Provider?

Reply:
yes. We initially thought we could keep it a bit more decoupled in the open source repo but alas, it didn't work out 🙃